### PR TITLE
Added functionality from $node entity to make generated entities revisionable

### PR DESCRIPTION
**Pull request by pdscti**

### CODE REVIEW

**Comment by jmolivas** [comment]
@pdscti Do you mind to sync changes to properly test this PR.

File: src/Command/Generate/EntityContentCommand.php [hunk 1]
```diff
@@ -38,6 +38,13 @@ protected function configure()
             InputOption::VALUE_NONE,
             $this->trans('commands.generate.entity.content.options.is-translatable')
         );
+
+        $this->addOption(
+          'is-revisionable',
```
Comment (jmolivas) [inline src/Command/Generate/EntityContentCommand.php]: Could you update this option as `revisionable` instead.

We are planning to update `is-translatable` as `translatable` only as well.

**Comment by pdscti** [comment]
Hi @jmolivas 

Rebased and amended the name of the option. Cheers for the feedback :)

**Comment by jmolivas** [comment]
@pdscti there is still an issue with PHPUnit on the build

https://travis-ci.org/hechoendrupal/DrupalConsole/jobs/149409889#L280

```
1.71s$ phpunit
PHPUnit 4.5.0 by Sebastian Bergmann and contributors.
Configuration read from /home/travis/build/hechoendrupal/DrupalConsole/phpunit.xml.dist
I................I....................E.......................... 65 / 69 ( 94%)
....
Time: 1.56 seconds, Memory: 34.75Mb
There was 1 error:
1) Drupal\Console\Test\Generator\EntityContentGeneratorTest::testGenerateEntityContent with data set #0 ('Foo', 'foo938160895', 'Bar', 'bar', 'admin/structure', 'true')
Missing argument 7 for Drupal\Console\Test\Generator\EntityContentGeneratorTest::testGenerateEntityContent()
/home/travis/build/hechoendrupal/DrupalConsole/Test/Generator/EntityContentGeneratorTest.php:36
                                                     
FAILURES!                                            
Tests: 69, Assertions: 105, Errors: 1, Incomplete: 2.
```

**Comment by pdscti** [comment]
Hi @jmolivas - thought I'd caught all of the changes in the tests, but looks like I missed a couple. Just trying again.

**Comment by pdscti** [comment]
@jmolivas Tests are passing: let me know if I've missed something though: PHP Unit it new to me.

**Comment by jmolivas** [comment]
@pdscti looking at this early next week.

**Comment by jmolivas** [comment]
@pdscti sorry for the late reply, but tagging the RC took longer that expected do you mind to sync your PR to have it merged.

EN translations got moved to this repo => https://github.com/hechoendrupal/drupal-console-en

The rest of the languages will be moved soon.

**Comment by pdscti** [comment]
@jmolivas I'm doing git merge upstream/master but it's telling me there are no changes to merge in. Am I missing something?

**Comment by pdscti** [comment]
@jmolivas That got it: I'd lost my remote origin somehow in all the excitement.

**Comment by jmolivas** [comment]
@pdscti looking at this today thanks for making the sync

**Comment by jmolivas** [comment]
@pdscti Thanks for your contribution, your PR was merged. This will be included on the next release.

**Pull request merged**